metpy/calc/thermo.py

```diff
@@ -52,6 +52,39 @@ def relative_humidity_from_dewpoint(temperature, dewpt):
     return (e / e_s)
 
 
+@exporter.export
+@check_units('[pressure]')
```

Comment:
I think this needs to be updated to include `[pressure]` for `reference_pressure` as well.